src/components/DateRangeFilter/DateRangeFilter.tsx

```diff
@@ -58,6 +90,11 @@ export const DateRangeFilter = ({ withRevokedRange, className }: DateRangeFilter
         <Button variant="outline">
           <ListFilter className="w-5 h-5" />
           <span className="ml-2">Filter</span>
+          {createdRange || updatedRange || revokedRange ? (
+            <span className="ml-2 bg-black/80 rounded-full text-white h-6 w-6 text-xs flex justify-center items-center">
+              +1
+            </span>
+          ) : null}
```

Comment:
After testing:
The `+1` content is not visually centered in the circle:
![image](https://github.com/user-attachments/assets/f3de719f-92bb-43eb-a033-4c8d8a81302e)

Reply:
Updated